index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java

```diff
@@ -39,26 +28,20 @@
 public final class Aggregations {
 
     public static final int MULTIPLE = 2;
+    public static final String ALL_AGGREGATIONS = "all";
     private static final CharSequence JSON_PATH_DELIMITER = ".";
 
     private Aggregations() {
     }
 
-    public static Map<String, Aggregation> generateAggregations(String username, String customer) {
+    public static Map<String, Aggregation> generateAggregations(String aggregationType, String username,
+                                                                String topLevelCristinOrg) {
```

Comment:
Looks good, but could you add an alternative to not to create aggregations? as in search-api -> aggregation=none

Reply:
We can add this later if we see the need 👍